### PR TITLE
tests/rotate: Pass integer parameters to hypothesis.st.integers

### PR DESCRIPTION
**Pull request by nbraud**
[Hypothesis #2117], released in v4.39.3, introduced tighter type annotations, causing `mypy` to reject int-representable floats as parameters to `st.integers`.

[Hypothesis #2117]: https://github.com/HypothesisWorks/hypothesis/pull/2117

### CODE REVIEW

File: tests/test_rotate.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def test_trig_invariance(angle: float, n: int):
     assert isclose(r_sin, n_sin, rel_to=[n / 1e9])
 
 
-@given(v=vectors(), angle=angles(), n=st.integers(min_value=0, max_value=1e5))
+@given(v=vectors(), angle=angles(), n=st.integers(min_value=0, max_value=10_000))
```
Comment (astronouth7303) [inline tests/test_rotate.py]: 10,000 is 1e4. Was this change intentional?

Comment (nbraud) [reply tests/test_rotate.py]: Oops, no; let me PR this back, thanks for the catch.
